content/docs/command-reference/repro.md

```diff
@@ -109,8 +109,10 @@ up-to-date and only execute the final stage.
   for example with subdirectories containing a separate pipeline that can be
   reproduced independently.
 
-- `-R`, `--recursive` - determines the stages to reproduce by searching each
-  target directory (if any) and their subdirectories.
+- `-R`, `--recursive` - expands arguments that `targets` accepts to determine
+  the stages to be reproduced, by searching each target directory and its
+  subdirectories. This option only has an effect if directory paths are given as
+  `targets`.
```

Comment:
First, this is a confusing text. It should be explained in simpler terms, shorter sentences, etc. Also, it has grammar errors.

Reply:
Secondly we're missing the whole point of #1632... Maybe you're confused by the term "expands". One thing is to expand a path (explore recursively) — that's not what the issue is about at all. Just forget about that term...

This is about the targets argument of dvc repro (and how -RP affect it). Again (from https://github.com/iterative/dvc.org/pull/1861#discussion_r511520908), "i.e. usually targets only accept "Stage or path to dvc.yaml or .dvc file to reproduce", right? It does not say that paths to directories are accepted. But these options (-RP) only work for directories, so what's going on?"

If you can answer that question, you'll be closer to understanding what's needed here.